pkg/mkimage-raw-efi/install

```diff
@@ -288,7 +288,8 @@ DISK_WITH_P3=""
 DISKS_TO_MERGE_COUNT=0
 PDEV_LIST=""
 MULTIPLE_DISKS=false
-INSTALL_ZFS=false
+# XXX for testing TBD revert
+INSTALL_ZFS=true
```

Comment:
Is this a draft review ?  Else INSTALL_ZFS is set always. Instead you can use grub parameter eve_install_zfs_with_raid_level

Reply:
This was a temp thing for testing. Removed.